_posts/2018-10-04-js-testing.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: post
+title:  "Javascript Projects Testing"
+date:   2018-10-04 12:00:00 -0400
+categories: software testing js javascript jest enzyme
+author: Tania Paiva
+---
+
+Should I test this? the answer is a BIG YES. Tests are your friends, and saviors sometimes (often).
+
+At Forward Financing we feel a big proud when our projects reach high test coverage, because thats our  
```

Comment:
This part could probably be better phrased as: 
"At Forward Financing, we feel proud when our projects reach high test coverage. It's our best insurance for..."